javascript/src/main/pages/Admin/Admin.js

```diff
@@ -1,21 +1,44 @@
 import React from "react";
-import { Table } from "react-bootstrap";
+import { Table, Button, Badge } from "react-bootstrap";
 import useSWR from "swr";
 import { useAuth0 } from "@auth0/auth0-react";
 import { fetchWithToken } from "main/utils/fetch";
 
 const Admin = () => {
   const { getAccessTokenSilently: getToken } = useAuth0();
   const { data: users } = useSWR(["/api/users", getToken], fetchWithToken);
-  const { data: admins } = useSWR(["/api/admins", getToken], fetchWithToken);
+  const { data: admins, mutate: mutateAdmins } = useSWR(["/api/admins", getToken], fetchWithToken);
 
-  const getRole = (user) => {
-    if (admins && admins.find(admin => {
-      return admin.email === user.email;
-    })) {
-      return "Admin";
+  const getAdmin = (user) => {
```

Comment:
I had to stare at this a long time to figure out what it's doing, and I'm still not entirely clear.

getAdmin is a function that takes a user and returns...  what?

Reply:
Given a user, I search the list of admins for their corresponding admin. This function either returns the matching admin or null.

I've renamed and re-parameterized this to make it more clear.